samples/machine-learning/parallel-half-moons/README.md

```diff
@@ -0,0 +1,52 @@
+---
+page_type: sample
+languages:
+- qsharp
+- csharp
+products:
+- qdk
+description: "This sample implements using the quantum machine learning library to train a sequential model on the half-moons dataset, parallelizing over target machines."
+---
+
+# Training sequential models with Q\#
+
+This sample uses Q# and the Microsoft.Quantum.MachineLearning library to train a simple sequential model.
+The model is trained on a half-moon dataset, loaded in C# using the System.Text.Json package, then preprocessed using Q#.
+
+
+## Prerequisites
+
+- The Microsoft [Quantum Development Kit](https://docs.microsoft.com/quantum/install-guide/).
+
+## Running the Sample
+
+This sample can be run in a number of different ways, depending on your preferred environment.
+
+<!-- TODO
```

Comment:
This should probably be removed, since for the parallelized sample going with a C# driver makes more sense.

Reply:
Good point, let me do a pass on comments / README on the new sample in general.